static/css/stylesheet.css

```diff
@@ -58,30 +58,35 @@ a{
   padding: 10px;
 }
 
-.navbar .navbar-inner{
-  padding:0
-}
-
 .navbar {
   border: none;
   border-radius: 0;
   margin: 0;
-  display: table;
   width: 100%;
   background-color: #2c3e50;
+  display: flex;
 }
 
-.navbar-nav .nav ul{
-  width: 100%;
+.navbar button {
+  border: none;
+  background-color: Transparent;
 }
 
-.navbar .nav li {
-  width: calc(100vw/5);
-  display: table-cell;
-  float: none;
+.navbar button div {
+  width: 20px;
```

Comment:
I kind of like the narrower navbar entries that you changed it to, but I think the dropdown options should be adjusted now:
(unnecessarily wide in comparison)
![image](https://user-images.githubusercontent.com/8433005/29240780-f16f9aae-7fa6-11e7-98de-c56c345c072a.png)